src/main/java/seedu/address/model/client/Client.java

```diff
@@ -54,12 +56,8 @@ public Address getAddress() {
         return address;
     }
 
-    public Meeting getMeeting() {
-        return meeting;
-    }
-
-    public void setMeeting(Meeting meeting) {
```

Comment:
Will require some way to add meetings to clients.

Reply:
Rather than returning the meeting purely (and allowing anyone to do anything with it), we can expose the method to add meetings via a proper method defined in this class (take in a meeting, add it into the arraylist so our arraylist is never exposed).